dev/com.ibm.ws.kernel.boot.core/src/com/ibm/ws/kernel/boot/internal/BootstrapConstants.java

```diff
@@ -99,6 +99,14 @@
     public static final String LOC_INTERNAL_LIB_DIR = "wlp.lib.dir";
 
     public static final String LOC_PROPERTY_INSTANCE_DIR = "wlp.user.dir";
+
+    /**
+     * Used (especially in the runnable JAR case at present) to specify that we should ignore WLP_USER_DIR set from the environment, and use
+     * something else as user dir. E.g. we might use: "${wlp.install.dir}/usr"
+     */
+    public static final String LOC_PROPERTY_INSTANCE_DIR_IGNORE_ENV = "wlp.user.dir.ignore.env";
```

Comment:
I think ignore would be better earlier since it would match how you read it. So wlp.ignore.user.dir.env makes more sense to me since  it says 'ignore user dir from environment'

Reply:
I could do that or:  `wlp.ignore.user.dir.from.env`

Reply:
Used `wlp.ignore.user.dir.from.env`